run-ovms.sh

```diff
@@ -40,53 +40,16 @@ fi
 cid_count=`ps aux | grep profile-launcher | grep -v grep | wc -l`
 
 #echo "barcode_disabled: $BARCODE_DISABLED, barcode_interval: $BARCODE_INTERVAL, ocr_interval: $OCR_INTERVAL, ocr_device: $OCR_DEVICE, ocr_disabled=$OCR_DISABLED, class_disabled=$CLASSIFICATION_DIABLED"
-pre_process=""
-if grep -q "rtsp" <<< "$INPUTSRC"; then
-	# rtsp
-	# todo pass depay info
-	inputsrc=$INPUTSRC
-	INPUTSRC_TYPE="RTSP"
-	pre_process="pre-process-backend=vaapi-surface-sharing -e pre-process-config=VAAPI_FAST_SCALE_LOAD_FACTOR=1"
-
-
-elif grep -q "file" <<< "$INPUTSRC"; then
-	# filesrc	
-	arrfilesrc=(${INPUTSRC//:/ })
-	# use vids since container maps a volume to this location based on sample-media folder
-	# TODO: need to pass demux/codec info
-	inputsrc="filesrc location=vids/"${arrfilesrc[1]}" ! qtdemux ! h264parse "
-	INPUTSRC_TYPE="FILE"
-	decode_type="vaapidecodebin"
-	pre_process="pre-process-backend=vaapi-surface-sharing -e pre-process-config=VAAPI_FAST_SCALE_LOAD_FACTOR=1"
-
+re='^[0-9]+$'
+if [[ "$INPUTSRC" != "rtsp*" && "$INPUTSRC" != "file*" && "$INPUTSRC" != *video* && "$INPUTSRC" =~ $re ]]; then
+	echo "assume realsense device..."
+	cameras=`ls /dev/vid* | while read line; do echo "--device=$line"; done`
+	TARGET_GPU_DEVICE=$TARGET_GPU_DEVICE" "$cameras
 elif grep -q "video" <<< "$INPUTSRC"; then
+	echo "assume video device..."
 	# v4l2src /dev/video*
 	# TODO need to pass stream info
```

Comment:
let's simplify this logic:
1) first check the video on the first if
2) if not, then check inputsrc to regular expression you already got; if matching the pattern then will assume it is real sense camera
3) else - then ignore...